src/core/types/addons.js

```diff
@@ -158,6 +158,11 @@ export type ExternalAddonType = {|
   weekly_downloads: number,
 |};
 
+export type PartialExternalAddonType = {|
+  ...ExternalAddonType,
+  current_version?: PartialExternalAddonVersionType,
```

Comment:
Maybe just remove this type?

It doesn't seem very useful since `ExternalAddonType` already has a `current_version` definition to accommodate both types:

https://github.com/mozilla/addons-frontend/blob/a6765efa490a36617f4c3d908c0c5e7f4cac8eb6/src/core/types/addons.js#L120

Alternatively, you could try fixing `ExternalAddonType`.

Reply:
I'm confused. Are you suggesting doing away with `PartialExternalAddonType`? That's what I'm using for the type in the recommendations reducer, which you commented on seems correct. If I get rid of that then these changes don't really accomplish much, do they? To me the whole point was to document the the `loadRecommendations` action gets `PartialExternalAddonType` objects as opposed to `ExternalAddonType` objects.